tick/optim/model/src/coxreg_partial_lik.h

```diff
@@ -9,8 +9,7 @@
 
 #include "model.h"
 
-
-class ModelCoxRegPartialLik : public Model {
+class DLL_PUBLIC ModelCoxRegPartialLik : public Model {
```

Comment:
the inference module file "inference_module_wrap.cpp" uses a number of model classes